public/index.html

```diff
@@ -10,6 +10,7 @@
     <!-- Import all js files -->
     <script type="text/javascript" src="./javascripts/gl-matrix-min.js"></script>
     <script type="text/javascript" src="./javascripts/main.js"></script>
+    <script type="text/javascript" src="./javascripts/teamImage.js"></script>
```

Comment:
To keep the convention, probably put this in a `teamImage` folder and use `'./javascripts/teamImage/teamImage.js'`.